src/main/java/school/hei/patrimoine/visualisation/swing/ihm/google/GoogleDocsSubmitScreen.java

```diff
@@ -1,151 +1,108 @@
 package school.hei.patrimoine.visualisation.swing.ihm.google;
 
 import static java.awt.BorderLayout.CENTER;
-import static java.awt.BorderLayout.NORTH;
-import static java.awt.Font.BOLD;
-import static java.awt.Toolkit.getDefaultToolkit;
-import static javax.swing.BoxLayout.Y_AXIS;
-import static javax.swing.JOptionPane.showMessageDialog;
 import static javax.swing.SwingUtilities.invokeLater;
-import static javax.swing.WindowConstants.EXIT_ON_CLOSE;
 
-import java.awt.Font;
+import java.awt.*;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.ExecutionException;
-import javax.swing.BoxLayout;
-import javax.swing.JButton;
-import javax.swing.JDialog;
-import javax.swing.JFrame;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
-import javax.swing.JTextField;
-import javax.swing.SwingConstants;
-import javax.swing.SwingWorker;
+import javax.swing.*;
+import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
-import school.hei.patrimoine.compiler.ClassNameExtractor;
-import school.hei.patrimoine.compiler.PatrimoineCompiler;
 import school.hei.patrimoine.google.GoogleApi;
-import school.hei.patrimoine.google.GoogleApi.GoogleAuthenticationDetails;
-import school.hei.patrimoine.google.GoogleDocsLinkIdParser;
-import school.hei.patrimoine.modele.Patrimoine;
-import school.hei.patrimoine.visualisation.swing.ihm.MainIHM;
 
 @Slf4j
 public class GoogleDocsSubmitScreen {
-  private final JFrame inputFrame; // Original frame for input
+  private final JFrame inputFrame;
   private final JPanel inputPanel;
-  private final List<JTextField> inputFields;
+  private final List<JTextArea> inputFields;
```

Comment:
you have only 1 JTextArea, there's no need for a List

Reply:
done